### DISCUSSION TITLE
New code4math community and Zulip chat

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#486] [Announcements]
@jamesdabbs @lyengulalp and I got to participate in https://aimath.org/workshops/upcoming/cyberinfrastructure/ this past week. One of several outcomes is that James and I are on the steering committee to develop a new professional organization:

https://code4math.org (website very much in its beginning stages!)

More interesting is our new Zulip, an open discussion forum for mathematicians interested in our shared sociotechnial infrastructure: https://code4math.zulipchat.com/ I encourage @prabau and @ccaruvana and any other folks interested in the pi-Base to join the conversation there, at least the #general and #pi-base streams.

We will continue to use the pi-base GitHub organization for pi-Base-specific discussions, issues, etc. etc. But I'm hopeful that the code4math community will be another avenue to expand our impact and grow our community.

**Comment by prabau** [comment]
fyi, trying to sign up at code4math.zulipchat.com and it says one needs an invitation to join.

**Reply by StevenClontz** [reply]
Strange, others have been able to join. I'll investigate

**Reply by StevenClontz** [reply]
Use this: https://code4math.zulipchat.com/join/p3r6km326kezmhdlvl477pzk/
